recipes/autovpn/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "autovpn" %}
+{% set version = "0.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/autovpn-{{ version }}.tar.gz
+  sha256: 6982ba40d85002e76ec6466896bac2a0f3d468f8aea4f8dc75c75c54ec51df6d
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - autovpn = autovpn:cli
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - click
+    - python >=3.7
+    - sortedcontainers
+    - toml
+
+test:
+  imports:
+    - autovpn
+  commands:
+    - pip check
+    - autovpn --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/corneliusroemer/autovpn
+  summary: A CLI wrapper for Cisco's Anyconnect VPN tool
+  dev_url: https://github.com/corneliusroemer/autovpn
+  license: MIT
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - 25161793
```

This should at least include your username:
```suggestion
    - corneliusroemer
```